CHANGELOG.md

```diff
@@ -98,7 +99,7 @@
 
 ### Bug fixes
 
-* Error occurring during test and connectedCheck of unit test example (#1934).
+* Error occurring during test and 2025.
```

Comment:
Why `connectedCheck of unit test example (#1934)` is deleted and `2025` is added?